src/controllers/expenses.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  getExpenseById,
+  updateExpenses,
+  removeExpenses,
+  createNewExpense,
+} = require('../services/expenses');
+
+const { getUserById } = require('../services/users');
+
+const addNew = async(req, res) => {
+  const { userId, title, amount, category, note } = req.body;
+
+  const foundUser = await getUserById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  const newExtense = await createNewExpense(
+    userId,
+    title,
+    amount,
+    category,
+    note
+  );
+
+  res.statusCode = 201;
+  res.send(newExtense);
+};
+
+const getOne = async(req, res) => {
+  const { expenseId } = req.params;
+
+  const foundExpens = await getExpenseById(expenseId);
+
+  if (!foundExpens) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.statusCode = 200;
```

Comment:
This line is redundant, delete it. And delete the same redundant lines all over the code.

200 status code - is the default one.

[Read more details here](https://stackoverflow.com/questions/53403078/express-is-it-necessary-to-respond-with-status-200)